yt_dlp/extractor/twitch.py

```diff
@@ -306,6 +348,26 @@ def _extract_info(info):
 
     @staticmethod
     def _extract_info_gql(info, item_id):
+        def moment_to_chapter(moment):
+            moment = moment.get('node')
+            if moment.get('type') != 'GAME_CHANGE':
+                # TODO: warning or debug log if found unknown moment type
+                return None  # TODO: or empty dict?
+
+            momentPosition = int_or_none(moment.get('positionMilliseconds'), 1000)
+            momentDuration = int_or_none(moment.get('durationMilliseconds'), 1000)
+
+            if momentPosition is None or momentDuration is None:
+                # TODO: warning or debug log about missing important data
```

Comment:
u can add a warning here

Reply:
how should I do that?
`report_warning` from utils is not accessible, because here I'm in a static method. should I remove the static decorator, or just write to stderr?

Reply:
hm.. idk why it is static method. Let me see if there was any reason. Let it be for now

Reply:
there is no reason for it to be static. You can convert it to a normal method and add the warning